src/libs/modals/modals/ModalConfirmStrategy/ModalConfirmStrategy.tsx

```diff
@@ -63,7 +65,7 @@ export const ModalConfirmStrategy: ModalFC<ModalConfirmStrategyData> = ({
         <IconTitleText
           variant={data?.variant}
           icon={data?.icon}
-          title={data?.title || ''}
+          title={data?.subtitle || ''}
```

Comment:
is this intentional? 

title vs. subtitle

Reply:
it was, but renamed it now to be more clear.